writer/writer_controller.go

```diff
@@ -16,11 +16,11 @@ const (
 )
 
 var (
-	MaxLogSize     = loadFromEnv("MAX_LOG_SIZE", 1<<32)                                     // default 4GB
-	MaxTimeAllowed = time.Duration(loadFromEnv("MAX_LOG_LIFETIME_MINS", 300)) * time.Minute // default 5 hours
+	maxLogSize        = loadFromEnv("MAX_LOG_BYTES", 1<<32)                                  // default 4GB
```

Comment:
similar to the other comment in the spade_edge CR this should be "loadIntFromEnv"